.travis.yml

```diff
@@ -5,7 +5,7 @@ dist: xenial
 dotnet: 2.2
 
 install:
-  - dotnet tool install -g coveralls.net --version 1.0.0
```

Comment:
Hello @NicolasSylverio, my friend.

I don't know where you found this 1.3.4 version of coveralls.net.

Our coveralls.net plugin is this: https://www.nuget.org/packages/coveralls.net and it have only version 1.0.0.

Just turn back to version 1.0.0 and the Travis-ci should back work.

Thanks for all your hep.

See you

Reply:
Also the pull from **develop** inside you **code_improvements-1** branch

Reply:
Hello @danilolutz.
I found a [Coveralls.net]( https://github.com/coveralls-net/coveralls.net), and I thought this might influence Travis, but when I saw that the pull request continued to break, I went back to 1.0.0 on commit 7855c40. 

I compiled and kept the error: 

`The command "$HOME/.dotnet/tools/csmacnz.Coveralls -i CoreZipCode.Tests/.coverage/coverage.opencover.xml --opencover --repoToken $COVERALLS_REPO_TOKEN --jobId $TRAVIS_JOB_ID --serviceName "travis-ci" --useRelativePaths" exited with 1.`